testing/test_driver.py

```diff
@@ -52,3 +52,45 @@ def test_driver_quit(selenium):
         'WARNING: Failed to gather log types: *'])
     outcomes = result.parseoutcomes()
     assert outcomes.get('failed') == 1
+
+
+def test_default_host_port(testdir):
+    host = 'localhost'
+    port = 4444
+    file_test = testdir.makepyfile("""
+        import pytest
+        @pytest.mark.nondestructive
+        def test_pass(driver_kwargs):
+            assert driver_kwargs['command_executor'] == 'http://{}:{}/wd/hub'
+    """.format(host, port))
```

Comment:
Rather than checking command_executor, can we not check host and port arguments directly?

Reply:
That was my initial implementation, but I got key-error on the dict and on inspection the only key it had with the host/port values were command_executor.